### PR TITLE
Bug13 border bottom

### PR DESCRIPTION
**Pull request by yewleongtoh**

### CODE REVIEW

File: scripts/html_to_pptx_converter_single_html.py [hunk 1]
```diff
@@ -339,6 +340,53 @@ def build_slide_from_data(prs: Presentation, slide_data: list[dict[any, any]]):
...
+            line_width = float(border_width.replace('px', '')) * 0.75
+            if line_width > 0:
+              left = Emu(px_to_emu(element['x_px']))
+              top = Emu(px_to_emu(element['y_px']))
+              width = Emu(px_to_emu(element['width_px']))
+              height = Emu(px_to_emu(element['height_px']))
+              padding_bottom = float(element['style']['paddingBottom'].replace('px', ''))
+              border_y = top + height + (padding_bottom * 0.25) # manual adjustment
+
+              # Convert color to RGB integers
```
Comment (teonimesic) [inline scripts/html_to_pptx_converter_single_html.py:362]: 362~376 could be made into a separate function, something like rgb_from_color as it could also be applied to other elements later on, since this isn't specific to border colors but rather all color attributes.

File: scripts/html_to_pptx_converter_single_html.py [hunk 2]
```diff
@@ -339,6 +340,53 @@ def build_slide_from_data(prs: Presentation, slide_data: list[dict[any, any]]):
         text_color = parse_rgb(element['style'].get('color'))
         if text_color:
           font.color.rgb = text_color
+
+        # --- Add border-bottom as a separate line ---
+        if 'borderBottom' in element['style']:
```
Comment (teonimesic) [inline scripts/html_to_pptx_converter_single_html.py:345]: the entirety of this if block should be extracted as a separate function, e.g.: element['style'] = convert_border(element['style'])

**Review by teonimesic — CHANGES_REQUESTED** [review]
Could do with some refactoring of code into functions to avoid clustering the build_slide_from_data function. Did not test manually if it works.

No unit tests added to help verify the code what it is intended, could add a few covering each edge case, e.g.: hex color conversion (with 6 and 3 digits), direct rgb values, default of black.

Could also add two screenshots, one for before and one for after to make it easy to see the difference
